File: python/insertion_sort.py
def insertion_sort(T:list[int],n:int)->list[int]:
 temp = T
 for i in range(1,n):
  for j in range(0,i+1):
   if(temp[i]<temp[j]):
    temp[i],temp[j] = temp[j],temp[i]
 return temp

File: c++/insertion_sort.cpp
#include<iostream>
#include<vector>
using namespace std;
void insertion_sort(vector<int>&T,int n)
{
 for(int i=1;i<n;i++)
 {
  for(int j=0;j<=i;j++)
  {
   if(T[j]>T[i])
    swap(T[i],T[j]);
  }
 }
}
int main()
{
 vector<int>T;
 T = {34, 7, 23, 32, 5, 62, 19, 14, 91, 3};
 insertion_sort(T,T.size());
 for(int i=0;i<T.size();i++)
 {
  cout<<T[i]<<" ";
 }
return 0;
}
